.ci/scripts/release/maven-release.sh

```diff
@@ -11,4 +11,5 @@ mvn -s ${MAVEN_SETTINGS_XML} release:prepare release:perform -B \
     -DpushChanges=${PUSH_CHANGES} \
     -DremoteTagging=${PUSH_CHANGES} \
     -DlocalCheckout=${SKIP_DEPLOY} \
-    -Darguments='--settings=${MAVEN_SETTINGS_XML} -DskipTests=true -Dgpg.passphrase="${GPG_PASS}" -Dskip.central.release=${SKIP_DEPLOY} -Dskip.camunda.release=${SKIP_DEPLOY} -Dzbctl.force -Dzbctl.rootDir=${ZBCTL_ROOT_DIR}'
+    -P!autoFormat \
```

Comment:
💭 I'm not sure whether -P!autoFormat is necessary on the release:prepare and release:perform steps. I imagine that these do no run the entire execution lifecycle, but rather only those 2 steps. In any case, it doesn't do any harm either.